.github/workflows/engine_indexer-connector_test.yml

```diff
@@ -39,12 +39,12 @@ jobs:
           vcpkgDirectory: '${{env.ENGINE_DIR}}/vcpkg'
           vcpkgGitCommitId: 'a42af01b72c28a8e1d7b48107b33e4f286a55ef6'
           vcpkgJsonGlob: '${{env.ENGINE_DIR}}/vcpkg.json'
-    
+
       - name: Configure CMake
         run: cmake --preset=${{env.BUILD_PRESET}} --no-warn-unused-cli -S ${{env.ENGINE_DIR}}
-  
+
       - name: Build
-        run: cmake --build ${{env.ENGINE_DIR}}/build --target indexer_connector_tool -j$(nproc)
+        run: cmake --build ${{env.ENGINE_DIR}}/build --target indexer_connector_tool wazuh-keystore -j$(nproc)
```

Comment:
I think we can revert these changes now that the indexer connector won't talk directly to the keystore tool right?

Reply:
You are right, I'll remove the this.